routes/index.js

```diff
@@ -15,10 +15,12 @@ router.post('/ingest/csvFile', upload.single('file'), async (req, res, next) =>
     const filePath = req.file.path;
     let results = [];
     fs.createReadStream(filePath)
-    .pipe(csv.parse({ headers: true }))
+    .pipe(csv.parse({ headers: ['id', 'phone'], renameHeaders: true}))
     .on('error', error => console.error(error))
     .on('data', row => {
-      results.push(row)
+      if(row.id !== '' && row.phone !== '') {
+        results.push(row)
+      }
```

Comment:
I'm assuming their headers will not match what we want
I'm also assuming that they'll have empty rows, based on the sample we were given - this will eliminate adding useless entries to the DB